projects/storefrontstyles/scss/components/misc/_icons.scss

```
@@ -1,3 +1,10 @@
+// Icons in an Right to Left context might require to be flipped.
+// Most icons will remain as-is, only icons that express a direction (i.e. caret-left)
+// are usually flipped.
+[dir='rtl'] cx-icon[flip-at='rtl'] {
```

For the feature to be RTL _inclusive_ I believe we should also add a second rule for flipping icons. That being said, I can hardly imagine any set of icons which are RTL-born and need to be flipped only in LTR layouts...

```suggestion
[dir='rtl'] cx-icon[flip-at='rtl'],
[dir='ltr'] cx-icon[flip-at='ltr'] {
```

---

true, i was too economic with the css selector, but it's so small so let's do it ;)